### DISCUSSION TITLE
I tried to add shadcn menubar to bubble menu but not working

### DISCUSSION THREAD

**Discussion by Milind-Jamnekar** [ueberdosis/tiptap#4710] [General Questions] [answered]
My `MenuBar` is in BubbleMenu component like this `<BubbleMenu>
				<MenuBar />
			</BubbleMenu>`

and currently my menubar is just example from shadcn lib which look like this 
`<Menubar>
<MenubarMenu>
				<MenubarTrigger
					onClick={() => editor.chain().focus().toggleBold().run()}
					disabled={!editor.can().chain().focus().toggleBold().run()}
				>
					File
				</MenubarTrigger>
				<MenubarContent>
					<MenubarPortal container={}>
						<MenubarItem>
							New Tab <MenubarShortcut>⌘T</MenubarShortcut>
						</MenubarItem>
						<MenubarItem>
							New Window <MenubarShortcut>⌘N</MenubarShortcut>
						</MenubarItem>
						<MenubarItem disabled>New Incognito Window</MenubarItem>
						<MenubarSeparator />
						<MenubarSub>
							<MenubarSubTrigger>Share</MenubarSubTrigger>
							<MenubarSubContent>
								<MenubarItem>Email link</MenubarItem>
								<MenubarItem>Messages</MenubarItem>
								<MenubarItem>Notes</MenubarItem>
							</MenubarSubContent>
						</MenubarSub>
						<MenubarSeparator />
						<MenubarItem>
							Print... <MenubarShortcut>⌘P</MenubarShortcut>
						</MenubarItem>
					</MenubarPortal>
				</MenubarContent>
			</MenubarMenu>
		<Menubar>
`

https://github.com/ueberdosis/tiptap/assets/58872755/bbd89d02-b2ad-4fc7-8a81-d836d0fd3b9d

well you can see here what's the problem. I am guessing when I click menu, bubble menu removing focus resulting removing entire menu as well from Dom and then menu drop down get rendered random in corner. I don't know how to solve it ? might need someone help

**Comment by liyasthomas** [comment]
I'm having the same issue. Did anyone find a fix?

**Reply by liyasthomas** [reply]
@Milind-Jamnekar have you solved this issue?

**Comment by nperez0111** [comment, marked as answer]
You need to keep the bubble menu open when the shadcn menu is open with the shouldShow prop on the bubble menu. On clicking anything in the menu you need to have it also toggle a Boolean to keep the bubble menu active since you've now blurred the editor.
